playbooks/roles/register_host/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Prepare inventory
+  become: yes
```

Comment:
There is `become: yes` here and in each playbook — is it required?

Reply:
yep, not required